etl_framework/config_mixins/DsnMixin.py

```diff
@@ -19,3 +20,11 @@ def set_dsn(self, dsn):
         """yup"""
 
         self.config[self.DSN_ATTR] = dsn
+
+    @check_config_attr_default_none
+    def set_dsn_from_dsn_type(self, etl_setup):
+        """
+        :summary: use the config to set the dsn
+        :param etl_setup(object): any object which has a DSN_TYPE attribute set
+        """
+        self.config[self.DSN_ATTR] = getattr(etl_setup, self.config[self.DSN_TYPE])
```

Comment:
@ianlofs can we use the set_dsn method instead of setting self.config directly?  Also, don't think we need @check_config_attr_default_none wrapper.